dev/com.ibm.websphere.appserver.features/visibility/public/mpRestClient-1.1/com.ibm.websphere.appserver.mpRestClient-1.1.feature

```diff
@@ -18,7 +18,8 @@ Subsystem-Name: MicroProfile Rest Client 1.1
  com.ibm.websphere.appserver.javax.annotation-1.2; ibm.tolerates:=1.3, \
  com.ibm.websphere.appserver.jaxrsClient-2.0; ibm.tolerates:=2.1, \
  com.ibm.websphere.appserver.jsonp-1.0; ibm.tolerates:=1.1, \
- com.ibm.websphere.appserver.org.reactivestreams.reactive-streams-1.0
+ com.ibm.websphere.appserver.org.reactivestreams.reactive-streams-1.0, \
```

Comment:
I thought this dependency was only added to RestClient 2.0

Reply:
Can't change it now, it's already GA.